packages/gitbook/src/components/SpaceLayout/SpaceLayout.tsx

```diff
@@ -116,6 +105,11 @@ export function SpaceLayout(props: {
                                 />
                             )
                         }
+                        innerHeader={
+                            sections || (!withTopHeader && site && spaces.length > 1) ? (
```

Comment:
can you extract it into a readable variable. Otherwise it's hard to read just by glancing over it